src/main/java/com/google/devtools/build/lib/rules/cpp/CcToolchainVariables.java

```diff
@@ -522,11 +537,16 @@ abstract static class VariableValueAdapter implements VariableValue {
     @Override
     public abstract boolean isTruthy();
 
+    @VisibleForTesting
```

Comment:
Linter says:
Methods should not be annotated with test visibility narrower than the visibility of methods they override. Consider removing the annotation to describe a less strict test visibility. It may also be appropriate to restrict the visibility of the super method.